IFC 4.3/SpatialDecomposition_1/UT_SpatialDecomposition_1.ifc

```diff
@@ -17,48 +17,43 @@ DATA;
+#18= IFCRELAGGREGATES('1MbrwbCg9EeRnmCIaa6zBR',#10,$,$,#11,(#4));
 #19= IFCCARTESIANPOINT((0.0,0.0,0.0));
 #20= IFCAXIS2PLACEMENT3D(#19,$,$);
 #21= IFCLOCALPLACEMENT(#3,#20);
-#22= IFCFACILITY('1_45ZlCCfBsef7TE9pDwQF',#10,'TrafficWayA',$,$,#21,$,$,.COMPLEX.);
-#23= IFCRELAGGREGATES('30xxoz_2r8jxHtElOoQuni',#10,$,$,#4,(#22,#40));
+#22= IFCFACILITY('2iR3QiEKT13xva7U_z3X$2',#10,'TrafficWayA',$,$,#21,$,$,.COMPLEX.);
+#23= IFCRELAGGREGATES('3gMREsTI573e1$2pf8yStl',#10,$,$,#4,(#22));
 #24= IFCCARTESIANPOINT((0.0,0.0,0.0));
 #25= IFCAXIS2PLACEMENT3D(#24,$,$);
 #26= IFCLOCALPLACEMENT(#21,#25);
-#27= IFCFACILITYPART('13E0fzdVb0QAY$KB0w5HS8',#10,'myRoadPart01','TrafficWayA -> Segment 1',$,#26,$,$,$,IFCROADPARTTYPEENUM(.ROADSEGMENT.),.LONGITUDINAL.);
-#28= IFCRELAGGREGATES('0SOOVX3lj4GOb7viYghKrH',#10,$,$,#22,(#27,#32,#36));
-#29= IFCCARTESIANPOINT((0.0,0.0,0.0));
-#30= IFCAXIS2PLACEMENT3D(#29,$,$);
-#31= IFCLOCALPLACEMENT(#21,#30);
-#32= IFCFACILITY('2Y2oRV7oT1m82v3uGf6jKJ',#10,'myBridge','TrafficWayA -> Segment 2',$,#31,$,$,$);
-#33= IFCCARTESIANPOINT((0.0,0.0,0.0));
-#34= IFCAXIS2PLACEMENT3D(#33,$,$);
-#35= IFCLOCALPLACEMENT(#21,#34);
-#36= IFCFACILITYPART('1ra6pWRbD18fEwOJ36SigM',#10,'myRoadPart02','TrafficWayA -> Segment 3',$,#35,$,$,$,IFCROADPARTTYPEENUM(.ROADSEGMENT.),.LONGITUDINAL.);
-#37= IFCCARTESIANPOINT((0.0,0.0,0.0));
-#38= IFCAXIS2PLACEMENT3D(#37,$,$);
-#39= IFCLOCALPLACEMENT(#3,#38);
-#40= IFCFACILITY('3QXXPR4Nz3jOkToYIaNHyu',#10,'River',$,$,#39,$,$,$);
-#41= IFCCARTESIANPOINT((0.0,0.0,0.0));
-#42= IFCAXIS2PLACEMENT3D(#41,$,$);
-#43= IFCLOCALPLACEMENT(#39,#42);
-#44= IFCFACILITYPART('1Ei_HGEiXAABHP9Hr0ESO4',#10,'myRiver','River that passes under the bridge',$,#43,$,$,$,IFCMARINEPARTTYPEENUM(.WATERFIELD.),.LONGITUDINAL.);
-#45= IFCRELAGGREGATES('275efIxLD8CRB2RceTtHRg',#10,$,$,#40,(#44));
+#27= IFCFACILITYPART('3358p6WkD51QxLDNAmC47A',#10,'myRoadPart01','TrafficWayA -> Segment 1',$,#26,$,$,$,IFCROADPARTTYPEENUM(.ROADSEGMENT.),.LONGITUDINAL.);
+#28= IFCRELAGGREGATES('3R1y780Mv2WOM1Ee_W8nTz',#10,$,$,#22,(#27,#29,#33));
+#29= IFCBRIDGE('2L4$ZHSpTDShVW6hQEDuP9',#10,'myBridge','TrafficWayA -> Segment 2',$,$,$,$,$,$);
+#30= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#31= IFCAXIS2PLACEMENT3D(#30,$,$);
+#32= IFCLOCALPLACEMENT(#21,#31);
+#33= IFCFACILITYPART('2lbWQSPrnEnhL25BweYKCs',#10,'myRoadPart02','TrafficWayA -> Segment 3',$,#32,$,$,$,IFCROADPARTTYPEENUM(.ROADSEGMENT.),.LONGITUDINAL.);
+#34= IFCMARINEFACILITY('0oKnsqXkz9Cg$qZ0du0Ioo',#10,'River',$,$,#3,$,$,$,.WATERWAY.);
+#35= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#36= IFCAXIS2PLACEMENT3D(#35,$,$);
+#37= IFCLOCALPLACEMENT(#3,#36);
+#38= IFCFACILITYPART('2UU0D29zr6Bf3s5j61okrf',#10,'myRiver','River that passes under the bridge',$,#37,$,$,$,IFCFACILITYPARTCOMMONTYPEENUM(.SEGMENT.),.LONGITUDINAL.);
+#40= IFCRELAGGREGATES('0vGUuSWij9YRiR$DyBdYjz',#10,$,$,#34,(#38));
+#41= IFCRELINTERFERESELEMENTS('1dAKzvmt5F2uOLIen2tEQL',#10,$,$,#29,#38,$,'PassesOver',.U.);
```

Comment:
see #68

Reply:
in my intended use in the standards development for this it is correct!